qt/python/mantidqtinterfaces/mantidqtinterfaces/dns_powder_elastic/plot/elastic_powder_plot_model.py

```diff
@@ -0,0 +1,110 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+
```

Comment:
Some of the copyright statements have a blank line breaking them up. Probably worth fixing because I know in the past we've had automated tools that modify the copyright statements, so it's good if they all match the same format.